src/app/debug/filter-side-drawer/filter-side-drawer.component.ts

```diff
@@ -52,35 +49,36 @@ export class FilterSideDrawerComponent implements OnDestroy, OnInit {
   }
 
   ngOnDestroy(): void {
-    this.unsubscribeAll();
+    this.genaralFilterSubscription?.unsubscribe();
```

Comment:
You can remove the ?

Reply:
I forgot to change it back, but now I have removed all the question marks